packages/next/src/server/config.ts

```diff
@@ -807,6 +807,18 @@ function assignDefaults(
     }
   }
 
+  if (result.experimental?.cacheLife) {
+    const defaultCacheLifeProfile = result.experimental.cacheLife['default']
+    if (!defaultCacheLifeProfile) {
+      throw new Error('No default cacheLife profile.')
+    } else {
+      if (defaultCacheLifeProfile.stale === undefined) {
+        const staticStaleTime = result.experimental.staleTimes?.static
+        defaultCacheLifeProfile.stale = staticStaleTime
```

Comment:
Do we need to check this for all profiles or just the default one since we fallback automatically?

Reply:
The thing that's special about stale is that it has a default as a separate config object.

The thing that's special about `"default"` is that it is used to seed the initial value of the `"use cache"` scope and then it can only be lowered from that implicitly. So all three need to exist on the default to define the implicit values.

https://github.com/vercel/next.js/blob/9877b0cfafc0a7b618292b4859827b26ecb25dec/packages/next/src/server/use-cache/use-cache-wrapper.ts#L186

Reply:
I also added `cacheLife.default.expire` defaulting to `expireTime`